src/systemdhook.c

```diff
@@ -18,6 +18,7 @@
 #include <linux/limits.h>
 #include <selinux/selinux.h>
 #include <yajl/yajl_tree.h>
+#include <stdbool.h>
```

Comment:
Yeah, that is fine. Just make sure that we enable c99 in our compilation if haven't already. Thanks!